python/lsst/ts/watcher/rules/mtdome_az_enabled.py

```diff
@@ -0,0 +1,106 @@
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MTDomeAzEnabled"]
+
+import typing
+
+from lsst.ts import salobj
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, BaseRule, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class MTDomeAzEnabled(BaseRule):
+    """Monitor the MTDome azEnabled event for any alarming states.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+    log : `logging.Logger`, optional
+        Parent logger.
+    """
+
+    def __init__(self, config, log=None):
+        remote_name = "MTDome"
+        remote_index = 0
+        remote_info_list = [
+            RemoteInfo(
+                name=remote_name,
+                index=remote_index,
+                callback_names=["evt_azEnabled"],
+                poll_names=[],
+            )
+        ]
+        super().__init__(
+            config=config,
+            name=f"MTDomeAzEnabled.{remote_name}:{remote_index}",
+            remote_info_list=remote_info_list,
+            log=log,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        # No schema necessary for this rule.
+        return None
+
+    def compute_alarm_severity(
+        self, data: salobj.BaseMsgType, **kwargs: typing.Any
+    ) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+              Message from the topic described by topic_callback.
+        **kwargs : `dict` [`str`, `typing.Any`]
+            Keyword arguments. If triggered by `TopicCallback` calling
+            `update_alarm_severity`, the arguments will be as follows:
+
+            * topic_callback : `TopicCallback`
+              Topic callback wrapper.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.idl.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+        severity, reason = NoneNoReason
+
+        if data.faultCode != "":
+            reason = data.faultCode
```

Comment:
There's a `status` attribute in this topic, which I think is an enumeration. I would suggest you add this information to the reason string, just to add more information. Maybe something like:

```
reason = f"Dome motion state {MotionState(data.state)!r}: {data.faultCode}."
```